elements/rh-card/docs/10-style.md

```diff
@@ -174,3 +182,90 @@ become thinner, the container padding decreases.
 <uxdot-example width-adjustment="360px">
   <img src="../card-spacing-mobile.svg" alt="Card spacing on mobile">
 </uxdot-example>
+
+## Variants
+
+### Promo
+
+Promos are available in four different configurations.
+
+#### Standard promo
+
+A Standard promo also has a subtle border, but it uses the `Lighter` or `Darker` surface color. It does not have an image slot and the call to action appears either to the left or right of the text.
+
+<uxdot-example>
+  <img src="../card-variants-promo-standard.svg" alt="Standard promo with placeholder text">
+</uxdot-example>
+
+#### Featured promo
+
+A Featured promo uses the `Lightest` or `Darkest` surface color with a subtle border. An image can appear on the left or right side of a promo, but a featured promo can also omit an image.
+
+<uxdot-example>
+  <img src="../card-variants-promo-featured.svg" alt="Featured promo with placeholder text and image">
+</uxdot-example>
```

Comment:
Use the following with correct `[values]` just to prep for the merge of PR later that fixes this very thing.
```Suggestion
<uxdot-example width-adjustment="[value]px">
  <img src="../card-variants-promo-featured.svg" 
      alt="Featured promo with placeholder text and image"
      width="[value]"
      height="[value]">
</uxdot-example>
```